server/manager/src/main/java/org/apache/accumulo/master/Master.java

```diff
@@ -1686,4 +1725,20 @@ public boolean isActiveService() {
     return masterInitialized.get();
   }
 
+  void initializeBalancer() {
+    tabletBalancer = Property.createInstanceFromPropertyName(getConfiguration(),
+        Property.MANAGER_TABLET_BALANCER, TabletBalancer.class, new DefaultLoadBalancer());
+    tabletBalancer.init(balancerEnvironment);
```

Comment:
Wondering if there is a threading problem and the balancer should only be set on the instance var after init().  Also if init throws an exception, may not want it set.  Wondering if tabletBalancer should be volatile.

```suggestion
    var localTabletBalancer = Property.createInstanceFromPropertyName(getConfiguration(),
        Property.MANAGER_TABLET_BALANCER, TabletBalancer.class, new DefaultLoadBalancer());
    localTabletBalancer.init(balancerEnvironment);
    tabletBalancer = localTabletBalancer;
```